package/src/components/CanvasZoom/CanvasZoom.jsx

```diff
@@ -67,8 +69,13 @@ class CanvasZoom extends Component {
   render = () => null
 }
 
-const WithCanvas = withCanvas(CanvasZoom)
+const ConnectedComponent =
+  withCanvas(
+    withHotKeys(
+      connect()(CanvasZoom)
```

Comment:
seems you should create an object with handler and register it using registerHandlers method from withCanvas ( as in CanvasDragger component)

Reply:
Sure, I have removed connect